build-tools/packages/build-cli/src/commands/release/report.ts

```diff
@@ -572,6 +581,9 @@ export default class ReleaseReportCommand extends ReleaseReportBaseCommand<
 			// Expand the release group to its constituent packages.
 			if (isReleaseGroup(pkgName)) {
 				for (const pkg of context.packagesInReleaseGroup(pkgName)) {
+					if (pkg.monoRepo?.releaseGroup !== "client") {
+						ranges.legacyCompat = ranges.caret;
+					}
```

Comment:
I think you should update the getRanges function instead:

https://github.com/microsoft/FluidFramework/blob/ecdf26a1d10490036cd95bc4d8bf37cb9b34175e/build-tools/packages/build-cli/src/library/release.ts#L83-L98

That function is intended to return all the known ranges for any version.

This is another example where the best mental model is to think of this as just an alternative representation of a range constraint - not as a separate version scheme. Given _any_ version, even an internal version, you should be able to generate a "legacy comapt" range representation for it.